doc/api/esm.md

```diff
@@ -299,6 +299,11 @@ The `type: 'json'` attribute is mandatory when importing JSON modules.
 
 ## Built-in modules
 
+> When importing built-in modules, all the named exports (i.e. properties of the module exports object) are populated even if they are not individually accessed.
+> This can make initial imports of built-in modules slightly slower compared to loading them with
+> `require()` or `process.getBuiltinModule()`, where the module exports object is evaluated immediately,
+> but some of its properties may only be initialized when first accessed individually.
```

Comment:
I believe this should not call out at the beginning of the "Built-in modules" section. This is not a warning.

I'd prefer that moving this to the end of the section if we really want to document it.